source/slang/core.meta.slang

```diff
@@ -1185,6 +1193,13 @@ extension matrix<T,N,M,L> : IFloat
     }
 }
 
+__generic<T:__BuiltinIntegerType, let R : int, let C : int, let L : int>
+extension matrix<T,R,C,L>
+{
+    __implicit_conversion(200)
```

Comment:
This needs to be turned in to a `kConversionCost_*` constant.

Reply:
I will add it.

Reply:
Hmm.. I just realized that the numbers have some meanings...
I will need to give higher value for int -> int8_t and int16_t.